pkg/issuer/acme/dns/akamai/akamai.go

```diff
@@ -165,6 +166,8 @@ func (a *DNSProvider) saveZoneData(domain string, data zoneData) error {
 }
 
 func (a *DNSProvider) makeRequest(req *http.Request) ([]byte, error) {
+	req.Header.Set("User-Agent", pkgutil.CertManagerUserAgent)
```

Comment:
cc @twz123, should I be setting a user-agent before or after signing the request? Does it matter?

Before seemed to make sense, but I don't know anything about akamai's stuff.

Reply:
headers are included in the data to sign, so it should probably be done before:

https://github.com/euank/cert-manager/blob/4bec3b9b06893d4aaee381256b2fe46c748b294d/pkg/issuer/acme/dns/akamai/edgegridauth.go#L115